.github/workflows/ci.yml

```
@@ -0,0 +1,15 @@
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Compile project
+    steps:
+      - uses: actions/checkout@v2
+      - run: touch config/.secrets.exs
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: '24'
+          elixir-version: '1.13.4'
+      - run: mix deps.get
+      - run: mix compile
```

It is common to add this in the CI to catch more errors:
```suggestion
      - run: mix compile --warnings-as-errors
```

In the future, you might also be interested in adding:
- `mix format --check-formatted` (built-in, to avoid committing non-formatted files)
- [dialyzer](https://hexdocs.pm/dialyxir/readme.html) for typechecking (the setup can be a bit more annoying)
- [credo](https://hexdocs.pm/credo/overview.html) if you want a linter (might be helpful to learn some best practices / avoid some pitfalls)